### ISSUE TITLE
List of things to make things

### ISSUE THREAD

**Issue by MatthewCroughan** [DoESLiverpool/somebody-should#1496]
**Issue to be closed a month after creation**

Can we discuss a shopping list of "Things we need in order to make things"

This list would include things like:

- Screws
- Bolts
- Washers
- Hinges
- Spring loaded mechanisms (detents, plungers, pogo pins).
- Standoffs

Things of this nature actually have many variables to them. You can get:

- locking  washers
- anchor bolts
- carriage bolts
- countersunk screws.

I have a budget of £100 and want to fill up the rotary cabinet next to dinky with various kinds of these "things" that we use to make things. So, I want to make a shopping list of all the things that we find useful and eventually make a wiki page on what each thing can be used for in the simplest 280 character limited sentence. E.G: "Lock washers are designed to be tightened to a required torque beneath an ordinary fastener."

Without this, people do not discover what they can do with the available hardware at DoES. I want to make it more easy to make things by discovery, without having to ask someone individually every single question of "what does this thing do?". I did this sort of discovery a few nights ago when I made a PC case out of wood on the laser cutter. I found all the kinds of nuts and bolts we had and used what was available. Though, I really wanted to use standoffs, I found 2 nuts + 1 bolt was sufficient to do the same task performed by a standoff. You could also 3d print a standoff, you could also use plastic tubing as a standoff. That's 4 ways of doing the same thing, that people at DoES would benefit from having a wiki page on, as well as links to purchase the hardware via the Wiki.

**Comment by helicalbytes** [comment]
Equipment wire -- reels of different colours.
How many different colours would make sense? What gauge etc.?

(Updated 18-Sep-2020)

Would these 2 sets work as a start to see how much they get used? They are just among the first that came up in a search, but they seem good value.

**Multi-strand wire: 6 colours x 25ft reels**
https://thepihut.com/products/hook-up-wire-spool-set-22awg-stranded-core-6-x-25ft

**Single core wire: 6 colours x 25ft reels -- suitable for breadboards**
https://thepihut.com/products/hook-up-wire-spool-set-22awg-solid-core-6-x-25-ft

They also sell the reels separately so we can replace them, add more colours or order more of the more frequently used colours.

**Comment by MatthewCroughan** [comment]
Adrian linked to https://github.com/FreeCAD/FreeCAD-library in the 20200917 Community Meeting
https://github.com/DoESLiverpool/somebody-should/wiki/Community-Meeting-20200917

**Comment by JackiePease** [comment]
12mm 24mm M3 bolts
M2 nuts, 12mm M2 bolts

**Comment by RiverMersey** [comment]
Does this issue/thread sound like a "reverse recipe book" for hardware?

Actually, this could become a great idea for hobbyists internationally!

Perhaps we could go as far as designing an interactive database where the user specifies quantities of standardised hardware that they have, and the database then suggests projects that meet that criteria?!

**Comment by MatthewCroughan** [comment]
@RiverMersey It is pretty much a reverse recipe book yes, that's a really good way of putting it. Just knowing what exists and roughly what function it serves in an item like a box will make you open the drawer and experiment with it. A glossy web interface that plugs into a list (maybe no real database format is even needed) would be fun yeah. 

We could make that happen when we've got a substantial list and understanding of all of these parts. Too far in the future for me right now though, I just want to fill that rotary cupboard up with parts people will use. Once I've done that, and we know what parts people are using, we can try and make a proof of concept.

**Comment by JackiePease** [comment]
M2, M3 spanners 
Heat shrink for LED strips
High shrink heat shrink for other stuff
Extra crimps for Molex kkk connectors

**Comment by MatthewCroughan** [comment]
@magman2112 This closes at the end of next week. Do you have any suggested bits and bobs that are useful for making any particular things?

**Comment by MatthewCroughan** [comment]
Everything requested has been purchased and is on its way to DoES. We should do another one of these in a few months in order to see whether we actually used what we got, and what else it is that people need.

Since this is the first one of these we've done, the result has been a bit fuzzy and lackluster. But the next one will hopefully have more input from people in the community.

![image](https://user-images.githubusercontent.com/26458780/97091033-a2c51b80-1630-11eb-91b0-cfd4c1b7f70e.png)
